drools-wb-screens/drools-wb-scenario-simulation-editor/drools-wb-scenario-simulation-editor-client/src/main/java/org/drools/workbench/screens/scenariosimulation/client/commands/ScenarioSimulationContext.java

```diff
@@ -110,6 +112,14 @@ public void setDataObjectFieldsMap(SortedMap<String, FactModelTree> dataObjectFi
         this.dataObjectFieldsMap = dataObjectFieldsMap;
     }
 
+    /**
+     * Set the names of already existing Data Objects/Instances, used inside updateHeaderValidation
+     * @param dataObjectsInstancesName
+     */
+    public void setDataObjectsInstancesName(Set<String> dataObjectsInstancesName) {
+        this.dataObjectsInstancesName = dataObjectsInstancesName;
```

Comment:
@danielezonca I checked. I confirm you that it's possible to create named instances in both grid with the current behaviour. Anyway, even if I revert it (putting again the `dataObjectsInstanceName' 'Set' in `ScenarioGridModel` class) the result it will be exactly the same, because every grid has its own `ScenarioGridModel` at the moment. Considering that our aim is to have one single `ScenarioSimulationContext` and one single `ScenarioGridModel` which will contains Data models for both scenario // background data, I think we can keep this change and modify the behaviour of the BackgroundGrid which will not consider this `Set` in the following PRs. Wdyt? @gitgabrio ?